packages/server/lib/request.js

```diff
@@ -469,14 +470,6 @@ module.exports = function (options = {}) {
       return createRetryingRequestStream(opts)
     },
 
-    contentTypeIsJson (response) {
-      // TODO: use https://github.com/jshttp/type-is for this
```

Comment:
This TODO has been fixed because I needed `typeis(response, ['text/*'])`. It has been removed in the last commit. I think it doesn't do any harm if it is left as fixed.

Reply:
Rolled back because of unit test failure.